packages/server/src/server.ts

```diff
@@ -95,5 +97,29 @@ export async function createServer(opts: ServerOptions, logger: LogType): Promis
     res.send(doc);
   });
 
+  /**
+   * @route GET /link/:tileSetId
+   *
+   * @example /link/ashburton-2023-0.1m
+   *
+   * @description If a tileset layer of the given id and 3857 projection exists, the server redirects the client to a Basemaps
+   * URL that is already zoomed to the extent of the layer. Otherwise, the server returns an 'HTTP 404 Not found' status code.
+   *
+   */
+  BasemapsServer.get(
```

Comment:
this needs to live in the lambda-tiler, all routes in lambda-tiler get deployed automatically into fastify with L87 https://github.com/linz/basemaps/pull/3354/files#diff-d6749baa2bf74a3f75f55f27e1f3c89512ad671f6dbec8db7e42ca6fa3377c38L87